src/calculateRentalCost.js

```diff
@@ -1,10 +1,20 @@
+'use strict';
+
 /**
  * @param {number} days
- *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50; 
+  } else if (days >= 3) {
+    totalCost -= 20; 
+  }
```

Comment:
Prefer `if` with `return` over `if else` to simplify later conditions. Since you're returning `totalCost` at the end, you can return it directly within the `if` blocks and avoid the `else if`.